src/elements/content-sidebar/activity-feed/annotations/AnnotationActivity.js

```diff
@@ -118,14 +119,14 @@ const AnnotationActivity = ({
                     <div>
                         <ActivityTimestamp date={createdAtTimestamp} />
                     </div>
-                    {isEditing && currentUser ? (
+                    {isEditing && isInputOpen && currentUser ? (
```

Comment:
If `isInputOpen` is false, then we shouldn't be showing the comment form.

Reply:
Do we care to maintain the `isInputOpen` state? Would basing the logic off of `isEditing` alone be sufficient? Doesn't seem like the `onFocus` behavior is needed

Reply:
Yep I've addressed this in the latest commit - we don't need isInputOpen logic here.